internal/controllermanager/prepare_controllers.go

```diff
@@ -244,8 +246,10 @@ func PrepareControllers(c *Config) (controllerinit.RunnerBuilder, error) { //nol
 		WithController(
 			jwtcachefiller.New(
 				c.AuthenticatorCache,
+				client.PinnipedConcierge,
```

Comment:
codecov is wrong here, the `.New()` would fail to instantiate if these were not passed in, and the code & tests would not compile.

Reply:
codecov is correct that this is not covered by unit tests, but that is intentional for this file. This is covered by integration tests.